core/src/main/java/uk/ac/bbsrc/tgac/miso/core/store/ExperimentStore.java

```diff
@@ -75,5 +76,5 @@
 
   Collection<Experiment> listByLibrary(long id) throws IOException;
 
-  Collection<Experiment> listByRun(long runId) throws IOException;
+  List<Experiment> listByRun(long runId) throws IOException;
```

Comment:
Should these changes be in a different PR?

Reply:
So, we actually use some kind of list for all storage, but return a mix of `Collection` and `List`. This mostly didn't matter because the security filters would was `<T> List<T> filterUnreadable(Collection<T>)`. With that gone, it was a more straight forward to update everything to use just `List`.